### PR TITLE
updated readme.md file

### PR DESCRIPTION
**Pull request by PriyaSoni164**
Closed readme.md file

### CODE REVIEW

**Review by NarendraSingh727 — APPROVED** [review]
Looks Ok

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 - Facilitator: @jessehouwing ⭐🚀
 
 ## Class Resources
-
+#adding comment to test
 - [GitHub for Developers Manual](https://githubtraining.github.io/training-manual/#/)
 - [Introduction to GitHub Flow](https://guides.github.com/introduction/flow/)
```
Comment (MKaiser2) [inline README.md:10]: just leaved a reply

**Review by MKaiser2 — APPROVED** [review]

**Comment by PriyaSoni164** [comment]
Updated readme.md

**Review by PriyaSoni164 — COMMENTED** [review]
updated readme

**Pull request merged**